dev/run-tests-jenkins.py

```diff
@@ -176,7 +177,8 @@ def main():
     build_display_name = os.environ["BUILD_DISPLAY_NAME"]
     build_url = os.environ["BUILD_URL"]
 
-    commit_url = "https://github.com/apache/spark/commit/" + ghprb_actual_commit
+    project_url = os.getenv("SPARK_PROJECT_URL", "https://github.com/apache/spark")
+    commit_url = project_url + "/commit/" + ghprb_actual_commit
```

Comment:
Looks good. BTW, do we need to add some dev documentation for this useful new configuration; `GITHUB_SERVER_API_URL` and `SPARK_PROJECT_URL`? Is this targeting too specific users?

Reply:
How about to add documentation at https://spark.apache.org/docs/latest/building-spark.html?

Reply:
+1